plugins/decl_hdf5/tests/compatibility_tests/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ set(RUNTEST_DIR "${CMAKE_CURRENT_LIST_DIR}/../../cmake/runtest-dir")
 
 set(ALL_TEST_NAMES array dataset group struct variables)
 if("${BUILD_HDF5_PARALLEL}")
-    list(APPEND ALL_TEST_NAMES mpi)
+    list(APPEND ALL_TEST_NAMES mpi mpi_independent)
```

Comment:
Ideally, we'd like to handle these differently.

Whatever the way the data is written, with MPI or not it should be compatible at the end. I believe they'd better fit at the same level as `HDF5_C_` and `PDI_C_` rather than the test name.

In this PR, I think what's there already can be used, but please make an issue about this.

Reply:
I create the issue